### PR TITLE
Fixed small spelling error

### PR DESCRIPTION
**Pull request by soenderby**
Changed "It's include automatically" to "It's included automatically"

### CODE REVIEW

File: www/src/pages/components/overlays.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default withLayout(function TooltipSection({ data }) {
           Overlays rely on a 3rd party library{' '}
           <a href="https://github.com/FezVrasta/react-popper">react-popper</a>,
           a tiny react wrapper around{' '}
-          <a href="https://popper.js.org/">Popper.js</a>. It's include
+          <a href="https://popper.js.org/">Popper.js</a>. It's included
           automatically with react-bootstrap, but you should reference the API
```
Comment (taion) [inline www/src/pages/components/overlays.js]:
```suggestion
           automatically with React-Bootstrap, but you should reference the API
```

File: www/src/pages/components/overlays.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default withLayout(function TooltipSection({ data }) {
           Overlays rely on a 3rd party library{' '}
           <a href="https://github.com/FezVrasta/react-popper">react-popper</a>,
           a tiny react wrapper around{' '}
```
Comment (taion) [inline www/src/pages/components/overlays.js]:
```suggestion
           a tiny React wrapper around{' '}
```

File: www/src/pages/components/overlays.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ export default withLayout(function TooltipSection({ data }) {
...
-          <a href="https://popper.js.org/">Popper.js</a>. It's include
+          <a href="https://popper.js.org/">Popper.js</a>. It's included
           automatically with react-bootstrap, but you should reference the API
           for more advanced use-cases.
```
Comment (taion) [inline www/src/pages/components/overlays.js]:
```suggestion
           for more advanced use cases.
```

File: www/src/pages/components/overlays.js [hunk 4]
```diff
@@ -50,7 +50,7 @@ export default withLayout(function TooltipSection({ data }) {
           Overlays rely on a 3rd party library{' '}
```
Comment (taion) [inline www/src/pages/components/overlays.js]:
```suggestion
           Overlays rely on the third-party library{' '}
```

**Comment by mxschmitt** [comment]
@soenderby please enable, that we also can make changes (by the Maintainers). So we can commit taion's changes.

**Comment by taion** [comment]
or accept the suggestions

**Comment by soenderby** [comment]
'Allow edits by maintainers' is enabled

File: www/src/pages/components/overlays.js [hunk 5]
```diff
@@ -47,12 +47,12 @@ export default withLayout(function TooltipSection({ data }) {
       <p>Things to know about the react-boostrap Overlay components.</p>
```
Comment (mxschmitt) [inline www/src/pages/components/overlays.js]:
```suggestion
      <p>Things to know about the React-Boostrap Overlay components.</p>
```

**Pull request merged**